Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'optimist'
```

Comment:
Could we add `kubeclient` and `filewatcher` here as well?

Reply:
Yep, I'll get them added.